docs/connector-development/connector-builder-ui/error-handling.md

```diff
@@ -0,0 +1,84 @@
+# Error Handling
```

Comment:
probably worth calling out that retries will be disabled on the connector builder. This is making me think maybe there should be a toggle to enable them if the developer wants to test the config.